tsdoc.json

```diff
@@ -17,5 +20,11 @@
             "syntaxKind": "block",
             "tagName": "@category"
         }
-    ]
-}
+    ],
+    "supportForTags": {
```

Comment:
what are all these for? and why the update to tsdoc config?

Reply:
I was chatting with @nbbeeken about the API extractor and that's what the updated config will look like. I'll remove from this PR.

Reply:
I originally thought we were getting errors with the experimental tag, we don't need this here, but would be a good improvement to add to get stricter validation on the permitted tags. It will pull in the settings supported by api-extractor. I can file a ticket, would be a quick win